podcasts/TranscriptsViewController.swift

```diff
@@ -91,38 +91,32 @@ class TranscriptsViewController: PlayerItemViewController {
     private func loadTranscript() {
         activityIndicatorView.startAnimating()
         Task.detached { [weak self] in
-            guard
-                let self,
-                let episode = self.playbackManager.currentEpisode(), let podcast = self.playbackManager.currentPodcast,
-                let transcripts = try? await ShowInfoCoordinator.shared.loadTranscripts(podcastUuid: podcast.uuid, episodeUuid: episode.uuid),
-                let transcript = transcripts.first else {
-                await self?.showResult(transcript: "", noTranscript: true, failedLoading: false)
+            guard let self else {
                 return
             }
-
-            guard
-                let transcriptURL = URL(string: transcript.url),
-                let transcriptText = try? String(contentsOf: transcriptURL)
-            else {
-                await self.showResult(transcript: "", noTranscript: false, failedLoading: true)
-                return
+            let transcriptManager = TranscriptLoader(playbackManager: self.playbackManager)
```

Comment:
Or maybe we could call it `TranscriptManager` :P 

Up to you!

Reply:
Changed to Transcript Manager [here](https://github.com/Automattic/pocket-casts-ios/pull/1859/commits/b4f979190241d47124b4c3affa3f0c4a6d247d70)